man/systemd.netdev.xml

```diff
@@ -298,6 +298,13 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>NetworkNamespace=</varname></term>
+          <listitem>
+            <para>Specifies the name for network namespace in which networkd will create net device.
+            If specified namespace doesn't exist then networkd will create it before creating the device.</para>
+          </listitem>
+        </varlistentry>
```

Comment:
From reading this I'm not clear what a "Network namespace" is... 

I mean.. from a kernel point of view, network namespaces don't have names at all. the iproute2 tools have a concept of named namespaces, but it's purely user-space IIUC.

This might also be a unit name in the "JoinNamespaceOf" sense... 

I think this deserves to be a bit more explicit.

Reply:
"Network namespace" context: http://man7.org/linux/man-pages/man7/namespaces.7.html

> I mean.. from a kernel point of view, network namespaces don't have names at all. 

You piqued my curiosity... http://man7.org/linux/man-pages/man8/ip-netns.8.html

So the names come from `/var/run/netns/`, and `ip netns identify` can be given a PID to reverse-lookup a name.

(Re-reading your comment it seems you knew all that, but anyway, links for others curious!)